repository-services/isajson-biosamples/Dockerfile

```diff
@@ -4,12 +4,19 @@ FROM gradle:jdk11 AS builder
 WORKDIR /app
 
 COPY . .
+
+WORKDIR /app/receipt
+RUN ./gradlew build
+RUN ./gradlew publishToMavenLocal
+
+WORKDIR /app/isajson-biosamples
 RUN ./gradlew build -x test -x spotlessApply --info --stacktrace "-Dorg.gradle.jvmargs=--add-exports jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED --add-exports jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED --add-exports jdk.compiler/com.sun.tools.javac.parser=ALL-UNNAMED --add-exports jdk.compiler/com.sun.tools.javac.tree=ALL-UNNAMED --add-exports jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED"
 
 # Use a lightweight base image to run the jar file
 FROM openjdk:11-jre-slim
+WORKDIR /app
```

Comment:
Since you are already in /app, the entrypoint should be `ENTRYPOINT ["java", "-jar", "app.jar"]` (without the /)